sql/moz-fx-data-shared-prod/subscription_platform_derived/apple_logical_subscriptions_history_v1/bigconfig.yml

```diff
@@ -0,0 +1,18 @@
+type: BIGCONFIG_FILE
+table_deployments:
+- collection:
+    name: 'Subscription Platform'
+    notification_channels:
+    - slack: '#de-bigeye-triage'
+    - email: 'phlee@mozilla.com, srose@mozilla.com'
```

Comment:
@phil-lee70 there are a lot of notifications for non-SubPlat ETLs in the `#de-bigeye-triage` channel, so personally I'd like to try the email notifications to get more targeted/noticeable alerts.

However, if you'd prefer to not get the email notifications we can remove your email from these configs (you'll just have to pay closer attention to the `#de-bigeye-triage` channel).